### DISCUSSION TITLE
Unity wallpaper webcam access prompts every time it starts

### DISCUSSION THREAD

**Discussion by Vantiveman** [rocksdanister/lively#1500] [Q&A]
Hi,
I know that this is undoubtedly a Windows 10 issue, and I have been living with it for ages, but I thought it was worth asking the question. I use a unity webcam as my windows background so I can see out into my garden while I work (I have three screens) but every time I start lively with this wallpaper selected I get a Webcam access warning prompt that I have to "OK" before it will start. This happens when I use admin permissions as well.
Are you aware of any way to prevent this warning? 
Thank you,
Al

**Comment by rocksdanister** [comment]
Is the Unity project exported as webpage? if its desktop application it should not ask again I think.
